app/views/errors.py

```diff
@@ -31,7 +31,7 @@ def add_cache_control(response):
 @errors_blueprint.app_errorhandler(NoTokenException)
 def unauthorized(error=None):
     log_exception(error)
-    return _render_error_page(401)
+    return render_theme_template('default', 'session-expired.html'), 401
```

Comment:
@hamishtaplin not very easily. i spent a day looking at this and as soon as flask know that the session has expired it removes any trace of it.